audio_service_web/LICENSE

```diff
@@ -0,0 +1 @@
+TODO: Add your license here.
```

Comment:
For the copyright notice, we can copy it from the other directories but I was wondering whether instead of "Ryan Heise and the project contributors", it might make more sense to have "Keagan Hilliard and the project contributors" if you are the principal author of the web implementation. I don't know enough about Copyright Law to know the exact implications of it, but I will support either choice.

Reply:
Now that this is a federated plugin, I also wonder whether "the project" is confusing, and whether it would be better to indicate specifically "audio_service_web". Hmm.